pkg/ctl/cmdutils/cmdutils.go

```diff
@@ -196,6 +196,11 @@ func AddCommonFlagsForGetCmd(fs *pflag.FlagSet, chunkSize *int, outputMode *prin
 	fs.StringVarP(outputMode, "output", "o", "table", "specifies the output format (valid option: table, json, yaml)")
 }
 
+// AddStringToStringVarPFlag is a wrapper that prefixes the description of the flag for consistency
+func AddStringToStringVarPFlag(fs *pflag.FlagSet, p *map[string]string, name, shorthand string, value map[string]string, usage string) {
+	fs.StringToStringVarP(p, name, shorthand, value, fmt.Sprintf(`A list of comma separated KV pairs "k1=v1,k2=v2": %s`, usage))
```

Comment:
nice, kind of nitpicking but I personally like the idea of the "format explanation" coming after the semantics of the option

Reply:
That's also fine but it will be more of a suffix format explanation. Fine by me too